config/config.local.ts

```diff
@@ -0,0 +1,25 @@
+import * as Config from "webpack-chain";
+
+import * as CommonConfig from "./config.common";
+import { EnvOptions } from "./types";
+
+function webpackConfig(options: EnvOptions = {}): Config {
+  // get the common configuration to start with
+  const config = CommonConfig.init(options);
+
+  const localPath = "/home/USER_NAME/.config/Screeps/scripts/127_0_0_1___21025/default/";
```

Comment:
If we move it to `credentials.json`, there's going to need to suddenly be a lot of code overhead to process it.

If they run a private instance of Screeps on a private server seperate from local, it wouldn't really be "local" anyway, right?  The whole point of the "local" config is: "this config is for when you *don't* want to upload"